### PR TITLE
force update to last sdk

### PR DESCRIPTION
**Pull request by thelittlefireman**
### Purpose of changes
Force update to last SDK to get the last emulator version. To try resolve some failed to run. #69 
Maybe we shoud update [https://github.com/appium/appium-docker-android/blob/master/Appium/Dockerfile](https://github.com/appium/appium-docker-android/blob/master/Appium/Dockerfile) line 86 to "ARG APPIUM_VERSION=1.7.2"
[https://github.com/appium/appium/blob/master/CHANGELOG.md](https://github.com/appium/appium/blob/master/CHANGELOG.md) => "Add support for MacOS 10.13"

### Types of changes
_Put an `x` in the boxes that apply_

- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by butomo1989** [comment]
Hi @thelittlefireman 

Thank you for the PR. I think that we are using the newest version of android SDK. Besides if you want to update the sdk, it should be done from [appium-docker-android](https://github.com/appium/appium-docker-android) side, because this project use it as base image and we can add the logics that if there is a new release of appium-docker-android then it will trigger the build in this project in travis and release a new version. I am not sure it is possible to do something like that in travis to be honest. 

In my opinion if we use your PR, we build the new image and release it today. and if tomorrow there is a new version of SDK, then the released image that we did still have the old version unless we release it again manually. what do you think?

**Comment by butomo1989** [comment]
and regarding issue #69 the emulator is out to date not the android SDK.

**Comment by thelittlefireman** [comment]
Hi,
thanks for the reply.

I'm wondering if the released Android-Sdk-Unix is out of date at release.

I checked and when you download  "sdk-tools-linux-3859397" you don't get the last emulator version: 
when you do a "sdkmanager --list" on your docker you get at the end : 
```
Available Updates:
  ID           | Installed | Available
  --------- | -------   | -------
  tools       | 26.0.1    | 26.1.1
  emulator | 27.1.7   | 27.1.10
done
```

So we need to check periodically if a new version is available

> then it will trigger the build in this project in travis and release a new version. I am not sure it is possible to do something like that in travis to be honest.`

Maybe with a webhook it could be done.

>and regarding issue #69 the emulator is out to date not the android SDK.

I don't thinks so : 
when you follow the instructions : 

> Your emulator is out of date, please update by launching Android Studio:
>  - Start Android Studio
>  - Select menu "Tools > Android > SDK Manager"
>  - Click "SDK Tools" tab
>  - Check "Android Emulator" checkbox
>  - Click "OK"

You get into this window : 
Emulator update
and you asked to update

That's can be done programmaticaly by :
```
echo y | sdkmanager "emulator" \
+    echo y | sdkmanager --update
```

File: docker/Emulator_x86 [hunk 1]
```diff
@@ -98,7 +98,8 @@ ENV ANDROID_VERSION=$ANDROID_VERSION \
 ENV PATH ${PATH}:${ANDROID_HOME}/build-tools
 RUN echo y | sdkmanager "platforms;android-${API_LEVEL}" && \
     echo y | sdkmanager "system-images;android-${API_LEVEL};${IMG_TYPE};${SYS_IMG}" && \
-    echo y | sdkmanager "emulator"
+    echo y | sdkmanager "emulator" \
```
Comment (butomo1989) [inline docker/Emulator_x86]: Could you put && before \ like previous line?

**Comment by butomo1989** [comment]
and also could copy paste the changes to Dockerfile for arm image: https://github.com/butomo1989/docker-android/blob/master/docker/Emulator_arm?

**Comment by butomo1989** [comment]
Thank you for the PR @thelittlefireman , it looks good for me. But could you do the small changes that I mentioned above? and I will merge it afterward.

File: docker/Emulator_x86 (hunk 1 above)
Comment (thelittlefireman) [reply docker/Emulator_x86]: oh you right my mistake  :s

**Comment by thelittlefireman** [comment]
@butomo1989 Just a quick question, i don't see in your SDK a way to accept all the licences of the sdk ? Did i miss something ? Because if i remeber well it's mandatory to download sdk ?

**Comment by butomo1989** [comment]
Hi @thelittlefireman , 

you just created same PR to appium-docker-android. appium-docker-android is used as base for docker image for this project. and if we release the new version of this project, it will use the latest emulator version. so I think it makes more sense to close this PR because it is done on appium-docker-android side. :)

and yes by doing echo y it will accept the license.

**Pull request closed without merging**